validators/fuji/fuji-versions.md

```diff
@@ -21,60 +21,65 @@ The rest of the versions were provided for docker containers and correspondent t
 
 **Important notes:**  
 
-1.  After docker installation make sure that the `live-restore` option
+1. Make sure you installed `1.27.4` docker-compose version. 
+2.  After docker installation make sure that the `live-restore` option
 is enabled in `/etc/docker/daemon.json`. See more info in the [docker docs](https://docs.docker.com/config/containers/live-restore/).  
 
-2.  If you have any issues you can save the logs using `skale logs dump` command.  
+3.  If you have any issues you can save the logs using `skale logs dump` command.  
 It's also useful to check logs from node-cli `skale cli logs` from docker plugin `/var/log/docker-lvmpy/lvmpy.log` if there are any issues.
 
-3.  You can install iptables-persistent using the following commands
+4.  You can install iptables-persistent using the following commands
     ```
     echo iptables-persistent iptables-persistent/autosave_v4 boolean true | sudo debconf-set-selections
     echo iptables-persistent iptables-persistent/autosave_v6 boolean true | sudo debconf-set-selections
     sudo apt install iptables-persistent -y
     ```
 
-4.  You should run skale commands using sudo
+5. You should run skale commands using sudo
+
+6. skale_sla container will be removed because it's deprecated for now.
+
+7. Logs from removed containers are placed inside `.skale/log/.removed_containers` 
+
+8. Before `skale node update` make sure that your attached storage is not mounted.
 
 ----
 
 ### TestNet Phase 3.1 Versions - SKALE chain creation
 Schains launch, IMA beta version
 
-**Validator CLI version**: 1.2.0-beta.2
+**Validator CLI version**: 1.2.0-beta.5
 
-**Node CLI version**: 1.1.0-beta.6
+**Node CLI version**: 1.1.0-beta.15
 
 **SGX version: sgxwallet**: 1.66.1-beta.0
 
 **SKALE Manager version**: 1.7.0-beta.0
```

Comment:
@gannakulikova Can you provide the latest SKALE Manager version to @badrogger

Reply:
`1.8.0-develop.8` new version

Reply:
or 1.8.0-beta.0
they are the same